context/AuthContextProvider.test.tsx

```diff
@@ -55,11 +63,36 @@ describe('AuthContextProvider', () => {
 
     mockCreateEmailPasswordSession.mockRejectedValue(mockError);
 
-    const error = await loginAccount({user, router, getUser});
+    const error = await loginAccount({ user, router, getUser });
+
+    expect(error).toEqual(mockError);
+    expect(toast.custom).toHaveBeenCalledWith(
+      <Alert variant={AlertVariants.Error} message="Something went wrong!" />,
```

Comment:
Please add a more descriptive message for this notification like you did for the logout test

Reply:
Most of the code that has been taken out was code that wasn't pushed up to alex's branch. so when I merged it just reverted it to what develop was.

Reply:
All of the login stuff in this code is alex's and anything that has to do with logout is stuff that I worked on.